content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst

```diff
@@ -1,23 +1,177 @@
+:guilabel:`Save` at the top of the form to save changes.
+
+.. tip::
+   For components that only apply for specific variants, choose which operations the components
+   should be consumed in. If the :guilabel:`Consumed in Operation` column is not immediately
+   visible, activate it from the additional options menu (three-dots icon, to the right of the
+   header row).
+
+Sell and manufacture variants of BoM products
+=============================================
+
+To sell and manufacture variants of :abbr:`BoM (bill of materials)` products to order, navigate to
+:menuselection:`Sales app --> Create` to create a new quotation.
+
+Sell a variant of a BoM product
+-------------------------------
+
+Once on the blank :guilabel:`Quotation` form, click the drop-down next to the :guilabel:`Customer`
+field to add a customer. Then, under the :guilabel:`Order Lines` tab, click :guilabel:`Add a
+product`, and select the previously-created :abbr:`BoM (bill of materials)` product with variants
+from the drop-down menu. Doing so reveals a :guilabel:`Configure a product` pop-up.
+
+From the pop-up, click the desired attribute options to configure the correct variant of the product
+to manufacture. Then, click the green :guilabel:`+` or :guilabel:`-` icons next to the `1` to change
+the quantity to sell and manufacture, if desired.
+
+.. image:: product_variants/product-variants-variant-popup.png
+   :align: center
+   :alt: Configure a product pop-up for choosing variant attributes.
+
+Once all the specifications have been chosen, click :guilabel:`Add`. This will change the pop-up to
+a second :guilabel:`Configure` pop-up, where available optional products will appear, if they have
+been created previously. Once ready, click :guilabel:`Confirm` to close the pop-up.
+
+Then, click :guilabel:`Save` to save all changes, and click :guilabel:`Confirm` at the top of the
+:guilabel:`Quotation` form to create and confirm a new sales order (SO).
+
+Manufacture a variant of a BoM product
+--------------------------------------
+
+Once the :abbr:`SO (sales order)` is confirmed, a :guilabel:`Manufacturing` smart button appears at
+the top of the :abbr:`SO (sales order)` form. Click the :guilabel:`Manufacturing` smart button to
+open the :guilabel:`Manufacturing Order` form.
+
+On this form, under the :guilabel:`Components` tab, the appropriate components for the chosen
+variant are listed, and depending on the variant, different components will be listed. To see any
+mandatory and optional :guilabel:`Operation` steps, click the :guilabel:`Work Orders` tab.
+
+To enter the tablet view work order screen, click the :guilabel:`tablet icon` to the right of the
+row for the desired operation to be completed. From here, click :guilabel:`Mark as Done` as the
+operation progresses to complete the operation steps. Alternatively, from the
+:guilabel:`Manufacturing Order` form, click :guilabel:`Mark as Done` at the top of the form to
+complete the order.
```

Comment:
Are we talking about the form view, not the tablet view?
If so, I'm not sure if wrapping `Manufacturing Order` with guilabel is the way to go since MO isn't explictly shown on the UI. `WH/MO/0008` is. In that case, maybe we should just call it "manufacturing order form" or "form view of the MO"

I reworded to say what the instruction does first, before the instruction itself. Feel free to accept/ignore this!

```suggestion
operation progresses to complete the operation steps. Alternatively, click the :guilabel:`Mark
as Done` at the top of the manufacturing order form to complete the order.
```